client/src/components/RatingsComponents/ReviewsSidebar/ReviewsSidebar.jsx

```diff
@@ -7,7 +7,7 @@ import PercentRecommended from './PercentRecommended.jsx';
 import RatingBreakdown from './RatingBreakdown.jsx';
 import RatingFactors from './RatingFactors.jsx';
 
-function ReviewsSidebar ({productId}) {
+function ReviewsSidebar ({productId, starsClicked, setStarsClicked}) {
   const [productMetaData, setProductMetaData] = useState({});
 
   useEffect(() => {
```

Comment:
Consider using params on this Fetch as well as removing the headers on Lines 18-20 as well as the portion of the url (Line 14) that would be covered by the params. TY!

Reply:
done and done